### PR TITLE
Update all doc for release 1.1

### LINKED ISSUE #175: Documentation to Updated CLI for 1.1
- Parents: #528 

---

## Checklist

- [ ] New Feature for the Service/Plugin (Vatz)
- [ ] Enhancement (Vatz)
- [x] others(etc. e.g, documentation, project policy management)

---

I'm always frustrated there's no instruction for 
- Development (VATZ, Plugins)

- Instruction or guideline to develop plugin that collect metrics so any new comer can install without any help 

TODOs
- [ ] Create documents under doc - How to develop VATZ and Plugins.

### PR DESCRIPTION
**Pull request by xellos00**
### 1. Type of change

Please delete options that are not relevant.

- [ ] New feature
- [x] Enhancement
- [ ] Bug/fix (non-breaking change which fixes an issue)
- [ ] others (anything other than above)

---

close #175 

**Summary**
Update documentation for release 1.1

---

### CODE REVIEW

File: docs/cli.md
Comment (gnongs) [inline docs/cli.md:105]: @xellos00 

It seems like a good idea to change it to VATZ for word uniformity.

File: docs/cli.md
Comment (gnongs) [inline docs/cli.md:137]: @xellos00 
It seems like a good idea to change it to VATZ for word uniformity.

File: docs/cli.md
Comment (gnongs) [inline docs/cli.md:316]: @xellos00 
In other examples, you've included something like <YOUR_VALOPER_ADDRESS>, but is there a reason why that's the only part that resembles the actual value?

File: docs/cli.md
Comment (xellos00) [reply docs/cli.md:47]: that's the global flag for log level when you execute CLI command. 
Is there any specific reason you ask --tract to be Capitalized?

File: docs/cli.md
Comment (xellos00) [reply docs/cli.md:105]: > @xellos00
> 
> It seems like a good idea to change it to VATZ for word uniformity.


Yeah, That's right. I agree!! I will make everything to `VATZ`

File: docs/cli.md
Comment (xellos00) [reply docs/cli.md:137]: Yeah, That's right. I agree!! I will make everything to `VATZ`

File: docs/cli.md
Comment (gnongs) [reply docs/cli.md:47]: @xellos00 
I was just trying to be consistent with the flag descriptions.

The other flags have lowercase letters in their descriptions, and this one has capitalized text.

Was I being too detailed..?😅

File: docs/cli.md
Comment (xellos00) [reply docs/cli.md:316]: I don't fully understand your point, but I think my answer can help your question if you are asking about why I used some value that is somewhat real alike only in this example. I'd wanted emphasize that every plugin arguments are different and they need to confirm exact arguments and its value from plugin's own info.  That's why I put the real alike address (which's fake address) in the value on example, and also attached sample plugins explanation for more details at line 311 ~ 316(which include plugin git address too) to let reader confirm for themselves.

File: docs/cli.md
Comment (xellos00) [reply docs/cli.md:47]: > @xellos00 I was just trying to be consistent with the flag descriptions.
> 
> The other flags have lowercase letters in their descriptions, and this one has capitalized text.
> 
> Was I being too detailed..?😅

No You are right, it's my bad.

File: docs/cli.md
Comment (gnongs) [reply docs/cli.md:316]: Oh, I missed checking that. I agree with what you said about that part.

**Review by gnongs — APPROVED** [review]
LGTM!

File: docs/cli.md
Comment (qkrybin) [reply docs/cli.md:49]: Here is an example config file:

File: docs/cli.md
Comment (qkrybin) [inline docs/cli.md:105]: You can start VATZ after setting up the default.yaml or your own config YAML file.

File: docs/cli.md
Comment (qkrybin) [inline docs/cli.md:128]: This will start VATZ with the default config created from the previous `vatz init` command. You can set the exact config file with the `--config` flag, such as:

File: docs/cli.md
Comment (qkrybin) [inline docs/cli.md:133]: an absolute path

File: docs/cli.md
Comment (qkrybin) [inline docs/cli.md:134]: by adding `--debug` or `--trace` flags

File: docs/cli.md
Comment (qkrybin) [inline docs/cli.md:311]: `--args`

File: docs/design.md
Comment (qkrybin) [inline docs/design.md:126]: The detailed design for VATZ v2 is currently under development and will be released in 2024. If you have any questions or comments regarding VATZ v2, please feel free to contact us.

File: docs/workflow.md
Comment (qkrybin) [inline docs/workflow.md:7]: must be passed

File: docs/workflow.md
Comment (qkrybin) [inline docs/workflow.md:23]: if you get approval from at least two reviewers.

File: docs/workflow.md
Comment (qkrybin) [inline docs/workflow.md:32]: all reviews are confirmed by at least two reviewers, and the PR will ~

File: docs/cli.md
Comment (kim201212) [inline docs/cli.md:8]: I think we need to unify whether the flag descriptions in all help are capitalized or lowercase, what do you think? It looks like this would require a code modification in `vatz/cmd`.

Comment (xellos00) [reply docs/cli.md:8]: @kim201212 
I updated all commands explanation to start with upper letter but couldn't update flag's `help` explanation to upper letter because only this case is already built with cobra library function and it starts with lower letter. I don't think I can fix it now unless I update cobra library or find other way to deal with. Let's leave it to later future tasks.

```
 ./vatz help         
Usage:
   [command]

Available Commands:
  completion  Generate the autocompletion script for the specified shell
  help        Help about any command
  init        Initialize VATZ
  plugin      Plugin commands
  start       Start VATZ
  stop        Stop VATZ
  version     VATZ Version

Flags:
      --debug   Enable debug mode on Log
  -h, --help    help for this command
      --trace   Enable trace mode on Log

```

Comment (kim201212) [reply docs/cli.md:8]: LGTM! Perfect!

**Review by kim201212 — APPROVED** [review]
LGTM!

File: docs/cli.md
Comment (qkrybin) [inline docs/cli.md:106]: You -> you

File: docs/cli.md
Comment (qkrybin) [inline docs/cli.md:110]: Please ensure the configuration setup is completed in either the `default.yaml` file or your custom configuration YAML file before initiating VATZ.

File: docs/cli.md
Comment (qkrybin) [inline docs/cli.md:134]: `vatz init`
or did you intentionally remove the backtick?

File: docs/cli.md
Comment (qkrybin) [inline docs/cli.md:140]: flag -> flags

File: docs/design.md
Comment (qkrybin) [inline docs/design.md:126]: currently in development

File: docs/cli.md
Comment (xellos00) [reply docs/cli.md:140]: it would be flag not flags, because using `or` between it.

File: docs/design.md
Comment (xellos00) [reply docs/design.md:126]: @qkrybin 
I think it's the pretty same phrase as 
currently in development = presently in the development stage
what do you think?

File: docs/cli.md
Comment (xellos00) [reply docs/cli.md:110]: seems more clear!!

File: docs/cli.md
Comment (xellos00) [reply docs/cli.md]: Agree!! fixed

File: docs/cli.md
Comment (xellos00) [reply docs/cli.md]: Agree!! that's more like it. fixed.

File: docs/cli.md
Comment (xellos00) [reply docs/cli.md]: updated!

File: docs/design.md
Comment (qkrybin) [reply docs/design.md:126]: agree

**Comment by qkrybin** [comment]
LGTM

**Pull request merged**
